scripts/histmakers/mz_dilepton.py

```diff
@@ -42,13 +42,13 @@
 mass_max = 120
 
 ewMassBins = theory_tools.make_ew_binning(mass = 91.1535, width = 2.4932, initialStep=0.010)
-
+ptV_binning = [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 17, 20, 23, 27, 32, 40, 54, 100] if not args.finePtBinning else range(60)
```

Comment:
Probably this should go together with all the other ptV_binnings in common, or have a different name

Reply:
Since this can have different values and is only needed in this file I think it makes sense to have it here. I Gave it a different name